ffs/manager/manager.go

```diff
@@ -31,7 +31,7 @@ var (
 	// recover its last configured default StorageConfig from the datastore.
 	zeroConfig = ffs.StorageConfig{
 		Hot: ffs.HotConfig{
-			Enabled: true,
+			Enabled: false,
```

Comment:
Decided with @asutula  to avoid disk usage explosion. 
If hte user want's HotStorage enabled, should be set explicitly on pushing a config.